frontend/src/views/WheresMyGene/common/types.ts

```diff
@@ -13,15 +14,25 @@ export interface CellType {
   name: string;
 }
 
+/**
+ * This object holds a gene and all the cell types that has this
+ * genetic expression.
+ */
 export interface RawGeneExpression {
   gene_name: string;
   cell_types: GeneExpression[];
 }
 
+/**
+ * This object describes the cell type id and the cell's genetic
+ * expression metadata
+ */
 export interface GeneExpression {
-  // cellTypeId
+  /** cellTypeId */
   id: string;
+  /** percentage of the cells that have this genetic expression */
   pc: number;
+  /** mean expression of the cells that have this genetic expression */
```

Comment:
Is this the mean expression level of the cells expressing this gene? or of all the cells (are zero values included in the average)

Reply:
CC: @atolopko-czi @pablo-gar please help here 🙏 Thank you!

Reply:
@MDunitz Since mean expression is averaging _rankit_ values, which are in the range (-inf, +inf), we filter out expr values where the original raw count is < 3.  In the code that was used to generate the prototype data, mean expression is computed as N/D, where N is number of cells with raw count >= 3, and D is the total number of cells across for the given cell_type, across all datasets.